documentation/modules/upgrading/proc-upgrading-consumers-streams-cooperative-rebalancing.adoc

```diff
@@ -1,10 +1,10 @@
 // Module included in the following assemblies:
 //
-// upgrading/assembly_upgrade-kafka-versions.adoc
+// upgrading/assembly-upgrade-kafka.adoc
 
 [id="proc-upgrading-consumers-streams-cooperative-rebalancing_{context}"]
 
-= Upgrading consumers and Kafka Streams applications to cooperative rebalancing
```

Comment:
I wonder if this section is better pulled out and sitting as a 7.3 under the assembly?

Reply:
Yes, good suggestion. I've moved it to the main assembly.